src/actions/users.js

```diff
@@ -610,6 +610,22 @@ export function getMe() {
     };
 }
 
+export function updateServiceTermsStatus(userId, serviceTermsId, accepted) {
```

Comment:
Why passing an userId, shouldn’t this be for the current user?

Reply:
Updated